src/model/jexchange/pool.provider.ts

```diff
@@ -0,0 +1,81 @@
+import { BigNumber, PoolTypes, SubgraphPoolBase } from "@trancport/aggregator";
+import { PoolNotActiveError, ProtocolProvider } from "../model.abstract.provider";
+import { Injectable } from "@nestjs/common";
+import { CachingService } from "src/common/caching/caching.service";
+import { formatFixed } from "src/utils/bignumber";
+import { Interaction, SmartContract } from "@multiversx/sdk-core/out";
+import { ApiConfigService } from "src/common/api-config/api.config.service";
+import { MvxApiService } from "src/common/mvx-communication/mvx.api.service";
+import { PoolConfig } from "../model.service";
+
+@Injectable()
+export class JexchangeProvider extends ProtocolProvider {
+    
+    constructor(
+        protected readonly cacheService: CachingService,
+        protected readonly apiConfigService: ApiConfigService,
+        protected readonly mvxApiService: MvxApiService,
+    ) {
+        super(apiConfigService, cacheService, mvxApiService);
+    }
+
+    getProtocolName(): string {
+        return "jexchange";
+    }
+    
+    getDirectoryName(): string {
+        return __dirname;
+    }
+
+    getSCExecuteInteraction(contract: SmartContract, tokenIn: string, tokenOut: string, minAmountOut: BigNumber): Interaction {
+        return contract.methods.swapTokensFixedInput([minAmountOut]);
+    }
+
+    getPoolType(): string {
+        return PoolTypes[PoolTypes.ProductConstant];
+    }
+
+    async loadPoolDataInternal(poolConfig: PoolConfig): Promise<SubgraphPoolBase> {
+        const firstTokenConfig = poolConfig.tokens[0];
+        const secondTokenConfig = poolConfig.tokens[1];
+        const [poolStatus] = await this.queryContractData(
+          poolConfig.address,
+          ['getStatus'],
+          [[]],
+        );
+
+        const totalFeePercent = (poolStatus.lp_fees as number) + (poolStatus.platform_fees as number);
```

Comment:
```
(poolStatus.lp_fees as number) + (poolStatus.platform_fees as number);
```
this line concentrate the string. Please change into:
```
const totalFeePercent = parseFloat(poolStatus.lp_fees) + parseFloat(poolStatus.platform_fees);
```

Reply:
done